portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,94 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
```

Comment:
Does bulma have a minified version? You may want to use that instead.

Reply:
Changed.